README.md

```diff
@@ -98,6 +98,16 @@ end
 ```
 
 
```

Comment:
mind making this one newline rather than two? I know it was there already, but that's weird